staging/src/k8s.io/apiserver/pkg/registry/rest/update.go

```diff
@@ -232,7 +232,7 @@ func (i *wrappedUpdatedObjectInfo) UpdatedObject(ctx genericapirequest.Context,
 }
 
 // AdmissionToValidateObjectUpdateFunc converts validating admission to a rest validate object update func
-func AdmissionToValidateObjectUpdateFunc(admit admission.Interface, staticAttributes admission.Attributes) ValidateObjectUpdateFunc {
+func AdmissionToValidateObjectUpdateFunc(admit admission.Interface, staticAttributes admission.Attributes, annotations map[string]string) ValidateObjectUpdateFunc {
```

Comment:
what are these annotations? As far as I see, the parameter is always nil.

Reply:
Same with https://github.com/kubernetes/kubernetes/pull/58143#discussion_r165251614